Src/StackifyLib.log4net/StackifyLib.log4net.csproj

```diff
@@ -13,6 +25,9 @@
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
     <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+    <AssemblyVersion>2.1.1.0</AssemblyVersion>
+    <FileVersion>2.1.1.0</FileVersion>
+    <Version>2.1.1-beta1</Version>
```

Comment:
rather than keeping this here, since `VersionPrefix` is already specified, I would specify the version suffix when you publish the artifacts